docs/deploy/configuration/env-vars/index.md

```diff
@@ -115,6 +115,20 @@ import APITable from '@site/src/components/APITable';
 | `SUM_WAIT_FOR_STARTUP` | If `true`, Weaviate waits for the `sum-transformers` module to start up before starting (default: `true`). (Available from `v1.25.27`, `v1.26.12`, `v1.27.7`) | `boolean` | `true` |
 | `GPT4ALL_WAIT_FOR_STARTUP` | If `true`, Weaviate waits for the `text2vec-gpt4all` module to start up before starting (default: `true`). (Available from `v1.25.27`, `v1.26.12`, `v1.27.7`) | `boolean` | `true` |
 | `TRANSFORMERS_WAIT_FOR_STARTUP` | If `true`, Weaviate waits for the `text2vec-transformers` module to start up before starting (default: `true`). (Available from `v1.25.27`, `v1.26.12`, `v1.27.7`) | `boolean` | `true` |
+| `ENABLE_MODULES` | Enable usage modules (GCS, S3, or both) | string | `usage-s3,usage-gcs` |
```

Comment:
There is another `ENABLE_MODULES ` variable in line 41, and we probably shouldn't have duplicate values here. The actual values of the variable `usage-s3` and `usage-gcs` could be listed in the `usage-modules.md` guide.